portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,30 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Raul's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
+    <div class="topbar">
+      <ul>
+          <li class="topbartext">Raul Cruz</li>
```

Comment:
nit: There's an inconsistency here in that sometimes you're just concatenating words for `class` names like `topbartext` and other times they're hyphen-separated words like `fact-btn` below. Can we pick one and make it consistent?

Reply:
Done.